src/Dfe.PlanTech.Web/Controllers/CookiesController.cs

```diff
@@ -1,33 +1,41 @@
 using Dfe.PlanTech.Application.Content.Queries;
+using Dfe.PlanTech.Application.Cookie.Interfaces;
 using Dfe.PlanTech.Domain.Content.Models;
 using Dfe.PlanTech.Web.Models;
-
-namespace Dfe.PlanTech.Web.Controllers;
-
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
+namespace Dfe.PlanTech.Web.Controllers;
+
 [Authorize]
```

Comment:
Can we remove the `[Authorize]` attribute please? Otherwise this won't work for unauthenticated users, even though they will see the banner etc.

Reply:
Removed